src/sage/combinat/integer_vector.py

```diff
@@ -782,21 +782,26 @@ def __contains__(self, x):
 
     def _unrank_helper(self, x, rtn):
         """
-        return an element at rank ``x``.
+        INPUT:
+
+        - ``x`` - a nonnegative integer
+        - ``rtn`` - a list of nonnegative integers
+
+        Return the element at rank ``x`` by iterating through all integer vectors beginning with ``rtn``.
```

Comment:
Sorry, slight miscommunication on my part: the sentence `Return ...` should be just below the `"""`.

Reply:
@mantepse Thank you for your kind words! These are my mistakes, and I'll try to minimize them next time.